src/main/java/de/szut/lf8_project/lf8/project/ProjectService.java

```diff
@@ -1,5 +1,8 @@
 package de.szut.lf8_project.lf8.project;
 
+import de.szut.lf8_project.exceptionHandling.InvalidDataException;
+import org.apache.catalina.User;
+import org.apache.catalina.connector.Response;
```

Comment:
import org.apache.catalina.User;
import org.apache.catalina.connector.Response;

can both be removed as they are not needed